ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -463,7 +468,10 @@ class FamilyReads extends React.PureComponent {
       </Segment.Group>
     ) : null
 
-    return React.createElement(layout, { variant, reads, showReads, ...props })
+    return React.createElement(
+      layout,
+      { variant, reads, showReads, updateReads: this.updateReads, familyGuid, ...props },
```

Comment:
I add the `updateReads` function to trigger the IGV display rather than change the `showReads` because the `showReads` shows a button we don't want.

Reply:
You should add a prop so that `showReads` is not created, if we aren't using it

Reply:
Good point. How about we add a prop of 'noTriggerButton'? When 'noTriggerButton' is provided, the `showReads` return a function rather than a button component.